test/critest.bats

```diff
@@ -19,6 +19,11 @@ function teardown() {
 	critest \
 		--runtime-endpoint "unix://${CRIO_SOCKET}" \
 		--image-endpoint "unix://${CRIO_SOCKET}" \
+		--parallel="$(nproc)" \
```

Comment:
Do we want to make it **-1** to keep some CPU for the rest of the things running and servicing I/O?

Reply:
Hm, I don't think the tests use that much CPU so I'd like to keep it simple :)